documentation/docs/concepts/decentralized_identifiers/update.mdx

```diff
@@ -46,363 +50,244 @@ methods allow them to be used by more than one verification relationship.
 />
 
-###  Creating Identity
+### Creating Identity
 
-The Example above starts by [creating an identity using the account](./create.mdx).
+The Example above starts by [creating an identity](./create.mdx).
 
 <Tabs groupId="code-examples">
 <TabItem value="Rust" label="Rust">
 
 ```rust
-let mut account: Account = Account::builder()
-    .storage(stronghold)
-    .create_identity(IdentitySetup::default())
-    .await?;
-```
-
-</TabItem>
-<TabItem value="node" label="Node.js">
-
-```js
-let builder = new AccountBuilder({
-    storage,
-});
-let account = await builder.createIdentity();
-```
-</TabItem>
-</Tabs>
+// Create a new client to interact with the IOTA ledger.
+let client: Client = Client::builder().with_primary_node(NETWORK_ENDPOINT, None)?.finish()?;
 
-This will create a DID document and publish it to the tangle.
+// Create a new secret manager backed by a Stronghold.
+let mut secret_manager: SecretManager = SecretManager::Stronghold(
+StrongholdSecretManager::builder()
+  .password("secure_password")
+  .build(random_stronghold_path())?,
+);
 
-```json
-{
-   "doc":{
-      "id":"did:iota:6T4PRHWp7bNsKaBWr1gVtUBQfLaxAKqKGAeJWFBZkMyf",
-      "capabilityInvocation":[
-         {
-            "id":"did:iota:6T4PRHWp7bNsKaBWr1gVtUBQfLaxAKqKGAeJWFBZkMyf#sign-0",
-            "controller":"did:iota:6T4PRHWp7bNsKaBWr1gVtUBQfLaxAKqKGAeJWFBZkMyf",
-            "type":"Ed25519VerificationKey2018",
-            "publicKeyMultibase":"z5k7vzMVuXXj8MJDcfzP2owvc8xKBA6BBsAkFf1GSNu2X"
-         }
-      ]
-   },
-   "meta":{
-      "created":"2022-04-13T09:27:48Z",
-      "updated":"2022-04-13T09:27:48Z"
-   }
-}
-```
-The created document only contains one verification method with [capabilityInvocation](https://www.w3.org/TR/did-core/#capability-invocation) relationship.
-This method is used to sign the DID Document for publication to the Tangle.
-The signature proves that the publisher of the document is in control over the capability invocation keys and is allowed to create, update or delete the DID Document.
-
-Any future updates to the DID Document in this example will be signed using this verification method. The Account will automatically sign each update with this method so individual
-updates don't have to be explicitly signed.
-
-Furthermore, it's possible to rotate a capability Invocation key. In this case, the Account will sign next update with a key which was valid in the previous state of the DID Document. Afterwards it will
- use the first (oldest) of the remaining capability invocation keys as a default signing method.
-Other capability invocation keys can still be explicitly specified to sign an update. These can be set in `PublishOptions`.
-
-
-Note that the Account does not allow removing all capability invocation keys.
+// Create a new DID in an Alias Output for us to modify.
+let (_, did): (Address, StardustDID) = create_did(&client, &mut secret_manager).await?;
 
-
-### Adding Verification Methods
```

Comment:
The example should be updated to demonstrate adding a verification method, since currently that operation is hidden by calling `create_did`, since it's no longer self-contained.